MBBSEmu/CPU/CPUCore.cs

```diff
@@ -2080,9 +2080,6 @@ private void Op_Cmp()
                 case 2:
                     Op_Sub_16();
                     return;
-                case 4:
```

Comment:
did we lose this case?

Reply:
Oh -- I stripped out the 32-bit branches I had started to write. Those snuck in because I was accidentally testing a MajorMUD version for Worldgroup 3.0 DOS.